suite-common/wallet-core/src/transactions/transactionReducer.ts

```diff
@@ -0,0 +1,145 @@
+import { accountsActions } from 'suite-common/wallet-core/src/index';
+
+import { Account, WalletAccountTransaction } from '@suite-common/wallet-types/libDev/src';
+import { findTransaction } from '@suite-common/wallet-utils/libDev/src';
+import { settingsCommonConfig } from '@suite-common/suite-config/libDev/src';
+import { createReducerWithExtraDeps } from '@suite-common/redux-utils/libDev/src';
+
+import { transactionActions } from './transactionActions';
+
+export interface TransactionState {
+    isLoading: boolean;
+    error: string | null;
+    // Key is accountHash and value is sparse array of fetched txs
+    transactions: { [key: string]: WalletAccountTransaction[] };
```

Comment:
please create `export type AccountHash = string` in suite-common/types/account and use it here instead